lib/src/test/java/space/taran/arklib/user/score/RootScoreStorageTests.kt

```diff
@@ -0,0 +1,60 @@
+package space.taran.arklib.user.score
+
+import dev.arkbuilders.arklib.user.score.RootScoreStorage
+import dev.arkbuilders.arklib.user.score.ScoreMonoid
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.GlobalScope
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import space.taran.arklib.randomResourceId
+import space.taran.arklib.randomScore
+import java.nio.file.Path
+
+class RootScoreStorageTests {
+
+    private val scope: CoroutineScope by lazy {
+        GlobalScope
+    }
+
+    private val pathMock: Path by lazy {
+        mockk()
+    }
+
+    private lateinit var rootScoreStorage: RootScoreStorage
+
+    @Before
+    fun setup() {
+        coEvery { pathMock.resolve(any<String>()) } returns pathMock
+        coEvery { pathMock.resolve(any<Path>()) } returns pathMock
+        rootScoreStorage = RootScoreStorage(scope, pathMock)
+    }
+
+    @Test
+    fun verifyGetScoreIfNoData() {
+        val resourceId = randomResourceId()
+        assertEquals(rootScoreStorage.getScore(resourceId), ScoreMonoid.neutral)
+    }
+
+    @Test
+    fun verifySetScore() {
+        val resourceId = randomResourceId()
+        val score = randomScore()
+        rootScoreStorage.setScore(resourceId, score)
+        assertEquals(score, rootScoreStorage.getScore(resourceId))
+    }
+
+    @Test
+    fun verifyRemove() {
```

Comment:
Hello team, it's a minor comment :) 

I would like to see more information in the test name like what we are expecting to see. Maybe also if it's not general agreement or convention (I did not found it in some Kotlin code conventions) to get rid of "verify" word as it's (maybe) unnecessary. 

For example, a test might be called "resourceRemovedById" or "verifyResourceRemoved" or even "removedResourceScoreNeutral"

What do you think?

Reply:
Good idea. Yes, it would be more clear and in the spirit of TDD. We should describe test cases in their names.
@tanaytandon12